### PR TITLE
add idiom 235 Decode base64

### PR DESCRIPTION
**Pull request by simonvonhackewitz**

### CODE REVIEW

File: src/idiom235ex.fz [hunk 1]
```diff
@@ -0,0 +1,13 @@
+ex235 is
+
+  s := "SGVsbG8gV29ybGQh"
+
+  data :=
+    match encodings.base64.decode_str s
```
Comment (michaellilltokiwa) [inline src/idiom235ex.fz:6]: I would prefer:
```
str := 
    match encodings.base64.decode_str s
      arr array u8 => String.type.from_bytes arr
      err error    => err.msg
```

Comment (simonvonhackewitz) [reply src/idiom235ex.fz:6]: According to the [idiom description](https://www.programming-idioms.org/idiom/235), the string should be decoded into a byte array.

Comment (michaellilltokiwa) [reply src/idiom235ex.fz:6]: Ah okay, hmm. Then another suggestion:
```
   match encodings.base64.decode_str s
      arr array u8 => arr
      err error    => panic "error while decoding: {err.msg}"
```

**Comment by michaellilltokiwa** [comment]
cool

**Pull request merged**